networking_aci/plugins/ml2/drivers/mech_aci/driver.py

```diff
@@ -265,6 +268,10 @@ def create_subnet_postcommit(self, context):
         self.rpc_notifier.create_subnet(context.current, external=external, address_scope_name=address_scope_name)
 
     def delete_subnet_postcommit(self, context):
+        if not CONF.ml2_aci.handle_all_l3_gateways or \
+                aci_const.CC_FABRIC_L3_GATEWAY_TAG in context.network.get('tags', []):
```

Comment:
why is this a `.get('tags', [])`, but the others access it as if it's always there i.e. `network['tags']`